### PR TITLE
Allows the detective scanner to scan objects up to 2 tiles away.

### PR DESCRIPTION
**Pull request by F-OS**
[BALANCE]
## About the PR 
Allows the detective scanner to scan objects up to 2 tiles away.
## Why's this needed?
This fixes the oversight where you put scanners into containers while allowing the detective to have a bit more latitude when conducting cases by scanning objects at a distance. I think this would increase quality of life for detective players, while not messing up game balance too much. This is a webeditor PR but the code was tested and confirmed to work in DM.
## Changelog

```changelog
(u)Fosstar
(+)The detective's scanner (not their PDA) can now scan objects up to two tiles away.
```

### CODE REVIEW

File: code/obj/item/device/scanners.dm [hunk 1]
```diff
@@ -165,6 +165,13 @@ that cannot be itched
 		user.show_text("No match found in security records.", "red")
 		return
 
+	pixelaction(atom/target, params, mob/user, reach)
+		if(get_dist(user, target) > 1 && get_dist(user, target) <= 2)
+			user.visible_message("<span class='notice'><b>[user]</b> takes a distant forensic scan of [target].</span>")
+			boutput(user, scan_forensic(target, visible = 1))
+			src.add_fingerprint(user)
+		return
```
Comment (ZeWaka) [inline code/obj/item/device/scanners.dm:173]: useless

File: code/obj/item/device/scanners.dm [hunk 2]
```diff
@@ -165,6 +165,12 @@ that cannot be itched
 		user.show_text("No match found in security records.", "red")
 		return
 
+	pixelaction(atom/target, params, mob/user, reach)
+		if(get_dist(user, target) > 1 && get_dist(user, target) <= 2)
```
Comment (ZeWaka) [inline code/obj/item/device/scanners.dm:169]: IN_RANGE

**Review by ZeWaka — CHANGES_REQUESTED** [review]

**Review by Tarmunora — CHANGES_REQUESTED** [review]
Could we get this *only* for a "detective's personal scanner" item? Would be nice to reinforce the feel that they are better equipped than standard security for forensics work.

**Review by Tarmunora — APPROVED** [review]

**Pull request merged**